### PR TITLE
add adjust camera

### PR DESCRIPTION
**Pull request by jessfraz**
needed for https://github.com/KittyCAD/engine/pull/1775

### CODE REVIEW

File: modeling-cmds/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "kittycad-modeling-cmds"
-version = "0.1.17"
+version = "0.2.0"
```
Comment (adamchalmers) [inline modeling-cmds/Cargo.toml:3]: Just use 0.1.18 and ignore the semver check linter

**Review by adamchalmers — CHANGES_REQUESTED** [review]

File: modeling-cmds/Cargo.toml (hunk 1 above)
Comment (adamchalmers) [inline modeling-cmds/Cargo.toml:3]:
```suggestion
version = "0.1.18"
```
Just use 0.1.18 and ignore the semver check linter

File: modeling-cmds/src/each_cmd.rs [hunk 2]
```diff
@@ -522,6 +522,7 @@ pub struct MouseClick {
 }
 
 /// Enable sketch mode on the given plane.
+/// If you want to sketch on a face, use `enable_sketch_mode` instead.
```
Comment (adamchalmers) [inline modeling-cmds/src/each_cmd.rs:525]: lol we've gotta change this, people are going to write a blog post making fun of us and they'll be right to. We can use Serde to make the underlying JSON string tags the same but use more useful names for the Rust types.

Comment (jessfraz) [reply modeling-cmds/src/each_cmd.rs:525]: I have an underlying issue for this, its a bigger change in that we need to evaluate where we use what in the app, converge one for a release, then we delete the old ya know or else we will break all app users https://github.com/KittyCAD/engine/issues/1774

**Comment by jessfraz** [comment]
going to merge since that was teh blocker and its fixed now, sorry i need this, I promise to follow the instructions to a tee for the release

**Pull request merged**